test/functional/wallet_importdescriptors.py

```diff
@@ -0,0 +1,293 @@
+#!/usr/bin/env python3
+# Copyright (c) 2019 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test the importdescriptors RPC.
+
+Test importdescriptors by generating keys on node0, importing the corresponding
+descriptors on node1 and then testing the address info for the different address
+variants.
+
+- `get_key()` is called to generate keys on node0 and return the privkeys,
+  pubkeys and all variants of scriptPubKey and address.
+- `test_importdesc()` is called to send an importdescriptors call to node1, test
+  success, and (if unsuccessful) test the error code and error message returned.
+- `test_address()` is called to call getaddressinfo for an address on node1
```

Comment:
`test_address` or another function could also test `getnewaddress` (in particular for an unranged descriptor, which imo should work exactly once)

Reply:
Disagree, on both counts.

Reply:
See [above](https://github.com/bitcoin/bitcoin/pull/16528#discussion_r399542742). It makes sense to disallow `getnewaddress` on unranged descriptors. And they can't be active (I assume that restriction is tested in the `importdescriptor` tests). So in that case there's nothing to test here.